NtupleProducer/python/HiggsTauTauProducer_106X.py

```diff
@@ -78,7 +78,7 @@
     else:
         process.GlobalTag.globaltag = '106X_mcRun2_asymptotic_preVFP_v11' # 2016 preVFP
   if YEAR == 2017:
-    process.GlobalTag.globaltag = '106X_mc2017_realistic_v9'             # 2017 MC
+    process.GlobalTag.globaltag = '106X_mc2017_realistic_v10'             # 2017 MC
```

Comment:
Did you produce some test events already with the new tag? If yes, I'm happy to merge